src/Lykke.Service.Operations/Modules/CqrsModule.cs

```diff
@@ -150,7 +150,13 @@ protected override void Load(ContainerBuilder builder)
                             .ListeningEvents(typeof(Bitcoin.Contracts.Events.CashoutCompletedEvent)).From(BitcoinBoundedContext.Name).On("events")
                             .ListeningEvents(typeof(Job.EthereumCore.Contracts.Cqrs.Events.CashoutCompletedEvent)).From(EthereumBoundedContext.Name).On("events")
                             .ListeningEvents(typeof(SolarCashOutCompletedEvent)).From("solarcoin").On("events")
-                            .ListeningEvents(typeof(CashoutCompletedEvent)).From(BlockchainCashoutProcessorBoundedContext.Name).On("events")
+                            .ListeningEvents(
+                                typeof(CashoutCompletedEvent), 
+                                typeof(CashoutsBatchCompletedEvent),
+                                typeof(CrossClientCashoutCompletedEvent),
+                                typeof(CashoutFailedEvent),
+                                typeof(CashoutsBatchFailedEvent))
```

Comment:
Don't see that subscription to OperationExecutionFailedEvent was removed. CashoutFailedEvent and CashoutsBatchFailedEvent replace it